controller/authenticate/twofactor.js

```diff
@@ -26,12 +26,10 @@ const sendSmsCode = async (phoneNumber) => {
 
```

Comment:
While you're in here, it looks like the documentation for these methods has drifted a bit if you don't mind either updating or perhaps removing the example objects.

Reply:
updated